containers/go/definition-manifest.json

```
@@ -1,14 +1,14 @@
 {
-	"variants": ["1.16", "1.15"],
-	"definitionVersion": "0.203.0",
+	"variants": ["1.16", "1.17"],
```

```suggestion
	"variants": ["1.17", "1.16"],
```

The order here matters. The zero index is assigned "latest", so put 1.17 there.